stdlib/std.jsonnet

```diff
@@ -1710,4 +1710,16 @@ limitations under the License.
   round(x):: std.floor(x + 0.5),
 
   isEmpty(str):: std.length(str) == 0,
+
+  remove(arr, elem):: [
```

Comment:
I think you can express this as something like:
```jsonnet
local indexes = std.find(arr, at);
if std.length(indexes) == 0 then
  return arr
else
  return std.removeAt(arr,indexes[0])
```

So it does not remove all of them

Reply:
Fixed